homework_3/barbhaiwala/solution.js

```diff
@@ -0,0 +1,83 @@
+function NaiveCalculator(equation)
+{
+    this.splitTokens = function()
+    {
+        return equation.split(' ');
```

Comment:
Also, because this line is so simple I wouldn't bother making a function. However, if you were, `equation` should be passed in as a variable instead of accessed from the parent scope. That way, it will be more testable.